### PR TITLE
Fix release-data i18n

### PR DESCRIPTION
**Pull request by Sea-n**
Not sure how to extract three single word.

Please give me some advice about better practices.

Current: https://kubernetes.io/releases/ (Hard-coded)
Preview: https://deploy-preview-35950--kubernetes-io-main-staging.netlify.app/releases/ (Not broken)
zh: https://deploy-preview-35950--kubernetes-io-main-staging.netlify.app/zh-cn/releases/ (Not translated yet)

Relates: #35932

### CODE REVIEW

File: data/i18n/en/en.toml [hunk 1]
```diff
@@ -234,6 +234,15 @@ other = "(released: "
 [release_date_format]
 other = "2006-01-02"
 
+[release_complete]
```
Comment (sftim) [inline data/i18n/en/en.toml:237]: I'd call this `release_full_details_initial_text`. We shouldn't name it after the English word.

File: data/i18n/en/en.toml [hunk 2]
```diff
@@ -234,6 +234,15 @@ other = "(released: "
...
+other = "Complete"
+
+[release_schedule]
+other = "Schedule"
+
+[release_changelog]
+other = "Changelog"
+
```
Comment (rolfedh) [inline data/i18n/en/en.toml:245]: Between lines 2 - 3, consider adding:
```
[and]
other = "and"
```

File: layouts/shortcodes/release-data.html [hunk 3]
```diff
@@ -33,10 +33,10 @@ <h3 id="release-v{{ $dataIdVersion }}">{{ $dataVersion }}</h3>
 </div>
 
 {{- end -}}
-<!-- not yet localized, mark as English -->
-<p lang="en">
-Complete {{ $dataVersion }} <a href="/releases/patch-releases/#{{ replace $dataVersion `.` `-` }}">Schedule</a>
-and <a href="https://git.k8s.io/kubernetes/CHANGELOG/CHANGELOG-{{ $dataVersion }}.md">Changelog</a>
+<p>
+{{ T "release_complete" }} {{ $dataVersion }}
+<a href="/releases/patch-releases/#{{ replace $dataVersion `.` `-` }}">{{ T "release_schedule" }}</a>{{ T "inline_list_separator" }}
```
Comment (rolfedh) [inline layouts/shortcodes/release-data.html:38]:
```suggestion
<a href="/releases/patch-releases/#{{ replace $dataVersion `.` `-` }}">{{ T "release_schedule" }}</a> {{ T "and" }}
```
For localization, should we preserve "and", like the original? Or does comma work better?

**Comment by rolfedh** [comment]
Fixes #35932

File: layouts/shortcodes/release-data.html (hunk 3 above)
Comment (sftim) [reply layouts/shortcodes/release-data.html:38]: It's a compromise. Another option is to make a format string that a localization team defines to build the sentence they need, with the `<a>` elements supplied as its input.

Let's do _something_ initially and then refine if we want to.

File: data/i18n/en/en.toml (hunk 2 above)
Comment (sftim) [reply data/i18n/en/en.toml:245]: I'm OK with this PR being a bit of a compromise. Remember that we can't assume the word order of target localizations.
We're _localizing_, not _translating_.

**Comment by bradtopol** [comment]
Assigning @sftim since he is already engaged on this.
/assign @sftim

**Comment by rolfedh** [comment]
By the way, I meant to thank you @Sea-n for quickly offering a solution! Excellent work!

**Comment by rolfedh** [comment]
@Sea-n 您如何看待这些评论？ 他们还好吗？
What do you think about these comments? Are they okay?

**Comment by rolfedh** [comment]
If you don't mind @Sea-n, I'm going to go ahead and create a new PR based on this one.

File: data/i18n/en/en.toml (hunk 1 above)
Comment (rolfedh) [reply data/i18n/en/en.toml:237]: In my PR clone of this one, I've _added_ an entry for `release_full_details_initial_text`. I'll deprecate the `release_complete` entry and advise the other localizations to update their `<lang>/<lang>.toml` files. Eventually, after a few releases, we'll remove the `release_complete` entry.

Comment (Sea-n) [reply data/i18n/en/en.toml:237]: Oh, sorry for missed this suggestion, I have changed accordingly.

**Comment by Sea-n** [comment]
I think it would be better to use a format string to enable the localization team to change the word order, but not sure how to do it in Hugo.

**Comment by rolfedh** [comment]
> tract three single word.
> 
> Please give me some advice.

- I don't know either. I think we should work to implement the current solution, and create a new issue for this requirement.
- This PR is based on the zh branch. When I look at the change history, it brings the change history from the zh branch into main. I think that might be a problem. Our contributor docs state that feature branches must be based on the branch (main) they will be merged into (main).

**Comment by Sea-n** [comment]
I didn't get it, isn't this PR based on (and merge to) `main` branch?

Could you provide some screenshots so that I can understand better.

**Comment by rolfedh** [comment]
Hi Sean. Sorry - I'm at work now, so I can't look into this until much later. Yesterday, when I did a test merge of your my first PR which was based on yours (Localize release-data i18n #36359) yesterday, I saw a lot of localized zh content in the changed that was unrelated to this PR. If you try `git log`, what do you see?

I recreated our work in a new PR that's based on main here: Fix release-data i18n #36375

**Comment by Sea-n** [comment]
Close as completed by another PR.
- #36375

**Pull request closed without merging**